### ISSUE TITLE
bulk_adding: if there's a validation error on the review page, parties are displayed as None afterwards

### ISSUE THREAD

**Issue by mhl** [DemocracyClub/yournextrepresentative#482]
If you forget to select something for someone on the review page, then you get returned to the same page after submitting the form. On this second time, the party names are displayed as "None" (though if you successfully submit the page the parties are recorded correctly).
![non-parties](https://user-images.githubusercontent.com/7907/38577235-3cc020a0-3cf8-11e8-8f04-12554a48f027.png)

**Comment by VirginiaDooley** [comment]
Currently blocked by https://github.com/DemocracyClub/yournextrepresentative/issues/1361

**Comment by VirginiaDooley** [comment]
Closing this issue as it's no longer relevant.
